
<launch>

  <!-- Visualisation (client side) -->
  <group ns="rtabmap">
    <node pkg="rtabmap" type="rtabmapviz" name="rtabmapviz" args="-d $(find rtabmap)/launch/config/rgbd_gui.ini" output="screen">
  	  <param name="subscribe_depth" type="bool" value="false"/>
      <param name="subscribe_laserScan" type="bool" value="false"/>
      <param name="queue_size" type="int" value="10"/>
      <param name="frame_id" type="string" value="base_footprint"/>
    
      <remap from="rgb/image" to="/camera/data_throttled_image"/>
      <remap from="depth/image" to="/camera/data_throttled_image_depth"/>
      <remap from="rgb/camera_info" to="/camera/data_throttled_camera_info"/>
      <remap from="scan" to="/base_scan"/>
      <remap from="odom" to="/base_controller/odom"/>
      
      <param name="rgb/image_transport" type="string" value="compressed"/>
      <param name="depth/image_transport" type="string" value="compressedDepth"/>
    </node>
  </group>
</launch>
